src/lib/bidi_markup.rb

```diff
@@ -0,0 +1,138 @@
+# Bidirectional Text: Left-to-right (Latin) and right-to-left (Arabic).
+#
+# See https://en.wikipedia.org/wiki/Bidirectional_text
+#
+# If you feel lost, try wrapping a piece of text that belongs together with
+# {.first_strong_isolate First Strong Isolate}.
+#
+# Quoting from the standard, Unicode TR 9, Section 6.3 Formatting,
+# where UPPER CASE simulates a RTL script:
+#
+# > The most common problematic case is that of neutrals on the
+# > boundary of an embedded language. This can be addressed by
+# > setting the level of the embedded text correctly. For example,
+# > with all the text at level 0 the following occurs:
+# >
+# >     Memory:  he said "I NEED WATER!", and expired.
+# >     Display: he said "RETAW DEEN I!", and expired.
+# >
+# > If the exclamation mark is to be part of the Arabic quotation,
+# > then the user can select the text I NEED WATER! and explicitly
+# > mark it as embedded Arabic, which produces the following result:
+# >
+# >     Memory:  he said "(RLI)I NEED WATER!(PDI)", and expired.
+# >     Display: he said "!RETAW DEEN I", and expired.
+module BidiMarkup
```

Comment:
Is this `yast2-storage-ng` specific problem? I think this should be moved to `yast2`, just in case we need the same solution elsewhere....

Reply:
In the end it looks like it might indeed be specific to storage, but the file is designed to be easily movable once we find out we need it.